ui/apps/dashboard/src/app/(dashboard)/env/[environmentSlug]/manage/[ingestKeys]/[keyID]/TransformEvent.tsx

```diff
@@ -182,7 +193,7 @@ export default function TransformEvents({ keyID, metadata, keyName }: FilterEven
         </TransformEditor>
       </div>
       <div className="mb-8 flex justify-end">
-        <Button kind="primary" type="submit" label="Save Transform Changes" />
+        <Button kind="primary" disabled={isDisabled} type="submit" label="Save Transform Changes" />
```

Comment:
Unrelated to this change, I had some weird git stuff happening and this disabled state popped back into my staged changed. I decided to keep this as I don't think that this should have been removed during the button refactor. cc @goodoldneon to confirm!